vitest.config.e2e.ts

```diff
@@ -16,6 +16,10 @@ export default defineConfig({
     testTimeout: timeout,
     hookTimeout: timeout,
     reporters: 'dot',
+    deps: {
+      // Prevent Vitest from running the workspace packages in Vite's SSR runtime
+      moduleDirectories: ['node_modules', 'packages'],
+    },
```

Comment:
Debugged this with Sheremet which Vitest used to auto externalize files that matches `**/dist/**`, but now not, which it would try to call `ssrLoadModule` on the Vite bundled code, which explodes the test (specifically dynamic imports doesn't work).

This config will auto-externalze packages with `**/node_modules/**` (default) and `**/packages/**` in the path.